main.go

```diff
@@ -152,14 +152,26 @@ func run(ctx context.Context) {
 }
 
 func runLoop(ctx context.Context) {
+	go func() {
+		metadataPollTicker := time.NewTicker(config.MetadataPollInterval())
+		for {
+			logger.Debugf("Polling metadata.")
```

Comment:
Don't debug log this as that will just completly fill logs. We need to have a few more log levels so we don't just jump from info to debug logs then we can add more verbose debug logging. I do want to add the Notice level for significant events and potentially only write those logs by default. Then we can use info for more logs and debug can be for really verbose logs. But we may also want a debug level.

Reply:
yeah..., i definitely think we need more  log level to make debugging easier and at the same time not filling up customers logs